src/main/java/mate/academy/spring/model/dto/request/OrderRequestDto.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.spring.model.dto.request;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import mate.academy.spring.model.Ticket;
```

Comment:
Instead of using the `Ticket` model directly, create a TicketRequestDto and use it here. This will help separate the domain model from the request data.